mssqlcli/mssql_cli.py

```diff
@@ -371,6 +375,11 @@ def run(self):
             print('Mail: sqlcli@microsoft.com')
             print('Home: http://github.com/dbcli/mssql-cli')
 
+        # if not in interactive mode, don't build CLI
+        if not self.interactive_mode:
```

Comment:
Does this by-pass interactive features, such as the pager?

Reply:
That's the intent—it runs `execute_command` and exits.

Reply:
Yes, but IIRC, execute_command pipe the results through a pager, which expects interactive key presses to page the results. I would expect us to have to add some logic around this. If we test with some larger output we can confirm the behavior.

Reply:
I see—I'll look into this.